update.php

```diff
@@ -1451,14 +1523,17 @@ function update_1163() {
 	$r = q("ALTER TABLE `item` ADD `network` char(32) NOT NULL");
 
 	set_config('system', 'maintenance', 0);
-	if(!$r)
-		return UPDATE_FAILED;
 
-	return UPDATE_SUCCESS;
+	if ($r) {
+		return UPDATE_SUCCESS;
+	}
+
+	return UPDATE_FAILED;
 }
 function update_1164() {
 	set_config('system', 'maintenance', 1);
 
+	/// @TODO If one update fails, should it continue?
```

Comment:
Now these changes are in an other branch, thanks to resetting hard to a specific commit:

````
$ git checkout rewrites/coding-convention
$ git checkout -b rewrites/coding-convention-splitX
$ git checkout rewrites/coding-convention
$ git reset --hard xyz123
````
This is what you can do to go back in a branch and still preserve the other changes.